llvm/test/tools/llvm-readobj/ELF/note-core-ntfile.test

```diff
@@ -3,6 +3,8 @@
 # RUN: yaml2obj %s -o %t.o
 # RUN: llvm-readelf --notes %t.o | FileCheck %s --check-prefix=GNU
 # RUN: llvm-readobj --notes %t.o | FileCheck %s --check-prefix=LLVM
+# RUN: llvm-readelf --elf-output-style=JSON --pretty-print --notes %t.o | FileCheck %s --check-prefix=JSON
+# RUN: llvm-readobj --elf-output-style=JSON --pretty-print --notes %t.o | FileCheck %s --check-prefix=JSON
```

Comment:
Could you check prior art in other tests, please, but I don't think there's any value in `llvm-readelf` and  `llvm-readobj` invocations where both use `--elf-output-style=JSON`.

Reply:
seems like the practice is to omit readelf